<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceListsForm</class>
 <widget class="QMainWindow" name="PriceListsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Прайс листы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата начала тура:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата окончания тура:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tourPriceLe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Стоимость тура</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPriceListBut">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBut">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
